test/e2e/gcs-emulator.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gcs-emulator
+  namespace: tekton-pipelines
+spec:
+  selector:
+    matchLabels:
+      run: gcs-emulator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: gcs-emulator
+    spec:
+      containers:
+      - name: gcs-emulator
+        image: quay.io/khrm/gcse-emulator:latest
```

Comment:
I build the image for `https://github.com/fullstorydev/emulators/tree/master/storage`. ATM, images aren't available.

Reply:
OK. Lets at least make that a comment in the yaml. Thanks.

Reply:
Added a comment for this.